packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -41,6 +102,14 @@ async function getLineBreakpointInfo(): Promise<{}> {
   return Promise.resolve(response);
 }
 
+export async function getApexTests(): Promise<{}> {
+  let response = {};
+  if (languageClient) {
+    response = await languageClient.sendRequest('apextest/getTestMethods');
```

Comment:
TODO: Need to reconcile this with the changes that are happening on the jorje side.